packages/introspect/src/generate.ts

```diff
@@ -496,78 +691,84 @@ function createWellKnownDefs(): Record<string, JSONSchema> {
-                'accurately processed in JavaScript/TypeScript',
+                'accurately processed in JavaScript/TypeScript.',
+                'To convert the string into a `Date` object,',
+                'use `new Date(timeString)` or `Date.parse(timeString)`',
             ].join('\n'),
         },
-        'pg_catalog.int2': {
-            id: 'pg_catalog.int2',
-            type: 'integer',
-            minimum: -32768,
-            maximum: 32767,
-            description: 'A PG int2',
-        },
-        'pg_catalog.int4': {
-            id: 'pg_catalog.int4',
-            type: 'integer',
-            minimum: -2147483648,
-            maximum: 2147483647,
-            description: 'A PG int4',
-        },
-        'pg_catalog.int8': {
-            id: 'pg_catalog.int8',
-            type: 'integer',
-            minimum: -9223372036854775808,
-            maximum: 9223372036854775807,
-            description: 'A PG int8',
-        },
-        'pg_catalog.float4': {
-            id: 'pg_catalog.float4',
-            type: 'number',
-            description: 'A PG float4',
-        },
-        'pg_catalog.float8': {
-            id: 'pg_catalog.float8',
-            type: 'number',
-            description: 'A PG float8',
+        'pg_catalog.tsquery': {
+            id: 'pg_catalog.tsquery',
+            type: 'string',
+            description: 'A PG tsquery',
         },
         'pg_catalog.tsvector': {
             id: 'pg_catalog.tsvector',
             type: 'string',
             description: 'A PG tsvector',
         },
-        'pg_catalog.bpchar': {
-            id: 'pg_catalog.bpchar',
-            type: 'string',
-            description: 'A PG bpchar',
-        },
-        'pg_catalog.bytea': {
-            id: 'pg_catalog.bytea',
+        'pg_catalog.txid_snapshot': {
+            id: 'pg_catalog.txid_snapshot',
             type: 'string',
-            description: 'A PG bytea',
+            description: 'A PG txid_snapshot',
         },
         'pg_catalog.uuid': {
             id: 'pg_catalog.uuid',
             type: 'string',
             format: 'uuid',
             description: 'A PG uuid',
         },
-        'pg_catalog.json': {
-            id: 'pg_catalog.json',
-            type: 'object',
-            description: 'A PG json',
+        'pg_catalog.varbit': {
+            id: 'pg_catalog.varbit',
+            type: 'string',
+            description: 'A PG varbit',
         },
-        'pg_catalog.jsonb': {
-            id: 'pg_catalog.jsonb',
-            type: 'object',
-            description: 'A PG jsonb',
+        'pg_catalog.varchar': {
+            id: 'pg_catalog.varchar',
+            type: 'string',
+            description: 'A PG varchar',
+        },
+        'pg_catalog.xml': {
+            id: 'pg_catalog.xml',
+            type: 'string',
+            description: 'A PG xml',
         },
     };
 }
+
+function createArrayWellKnownDefs(
```

Comment:
Do we really need this? This seems like it just creates an explosion of possible types.